src/main/java/frc/team2412/robot/Robot.java

```diff
@@ -101,10 +104,20 @@ public void robotInit() {
 
 	@Override
 	public void testInit() {
-		CommandScheduler.getInstance()
-				.schedule(
-						new diagnosticSequentialCommand(
-								subsystems.intakeSubsystem, subsystems.launcherSubsystem));
+		if (SubsystemConstants.INTAKE_ENABLED && SubsystemConstants.LAUNCHER_ENABLED) {
+			CommandScheduler.getInstance()
+					.schedule(
+							new diagnosticSequentialCommand(
+									subsystems.intakeSubsystem, subsystems.launcherSubsystem));
+		} else if (SubsystemConstants.INTAKE_ENABLED) {
```

Comment:
What is the use case for these being if else?  This requires subsystems to be disabled / enabled and then a full recompile to run diagnostic commands for different subsystems. 
 Also there doesn't seem to be a way to run diagnostics for multiple systems since only one condition can be satisfied.